week-1-ts-ecosystem/w01e02-enter-the-world-of-typescript/src/types/types.ts

```diff
@@ -0,0 +1,13 @@
+type PriceType = {
+  value: number;
+  currency: string;
+};
+
+type ItemType = {
+  name: string;
+  amount: number;
+  unit: string;
+  price: PriceType;
+};
+
+export type { PriceType, ItemType };
```

Comment:
Tutaj dokładnie o to chodziło żeby dać miejsce dla tych "ważnych" z perspektywy naszych danych typów.
A pozostałe to tak jak zauważyłeś `PropTypes`

Ja bym poszedł krok dalej i dał te Typy do oddzielnych plików w folderze `model`.